guides/installing-python.md

```diff
@@ -0,0 +1,89 @@
+# Installing and using Python 2 and 3
+
+## Remove Homebrew Python, virtualenvwrapper, and variables
+
+Before installing and configuring pyenv, uninstall all old versions of Python that might be installed via Homebrew and virtualenvwrapper, and remove any varaibles from your shell's dotfiles (`.bashrc` for bash users, `.zshrc` for zsh users):
+
+```shell
+brew uninstall python
+brew cleanup python
+```
+
+Example of variables you may find that need to be removed:
+
+```shell
+export WORKON_HOME=$HOME/.virtualenvs
```

Comment:
I don't *think* we need to have folks remove this line only to add it back below.